app/assets/javascripts/templates/project_view.ejs

```diff
@@ -13,7 +13,7 @@
         <td data-column-view="done"></td>
       <% } %>
       <td style="display: none;" data-column-view="epic" data-hideable="false"></td>
-      <td style="display: none;" data-column-view="history" data-hideable="false"></td>
+      <td style="display: none;" data-history-view="history" data-hideable="false"></td>
```

Comment:
Why is this change necessary? It doesn't make sense for me to have the value `history` in a data-attribute called `data-history`.